src/main/kotlin/no/nav/familie/ef/sak/repository/domain/Behandling.kt

```diff
@@ -22,7 +23,10 @@ data class Behandling(@Id
 
                       @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
                       val sporbar: Sporbar = Sporbar(),
-                      val resultat: BehandlingResultat)
+                      var resultat: BehandlingResultat) {
+
+    fun kanAnnulleres(): Boolean = !status.behandlingErLåstForVidereRedigering() && type == BehandlingType.BLANKETT
```

Comment:
👍

Reply:
Og (bare for at jeg skal henge med i svingene) behandlingErLåstForVidereRedigering - betyr vel at bare saksbehandler 1 kan annulere? Dersom vi er inne i totrinn må vi vel underkjenne - deretter annulere?

Reply:
Korrekt.